src/plots/gl3d/scene.js

```diff
@@ -97,7 +97,7 @@ proto.tryCreatePlot = function() {
         gl: scene.gl,
         glOptions: {
             preserveDrawingBuffer: isMobile,
-            premultipliedAlpha: true,
+            premultipliedAlpha: false,
```

Comment:
Hmm. Does this lead to any changes in performance ?

Reply:
Good question. It shouldn't. Anyway I would double check.

Reply:
We decided not to change this config in this PR for now: https://github.com/plotly/plotly.js/pull/4360/commits/ade22a674393762b8a7262c368acb5e0fae0661f.